Support/en.lproj/Localizable.strings

```diff
@@ -263,7 +263,10 @@
 "enum.navigation_item.new" = "New";
 "enum.navigation_item.downloads" = "Downloads";
 "enum.navigation_item.settings" = "Settings";
+"enum.navigation_item.donations" = "Support Kiwix";
 "enum.search_result_snippet_mode.disabled" = "Disabled";
 "enum.search_result_snippet_mode.paragraph" = "First Paragraph";
 "enum.search_result_snippet_mode.sentence" = "First Sentence";
 "enum.search_result_snippet_mode.matches" = "Matches";
+
+"donations.url" = "https://kiwix.org/en/get-involved/";
```

Comment:
I am a bit torn on where to put this URL, to be honest.
We have this page on the website in a couple of languages, but that's not 100% mappable to iOS/macOS supported languages.

Things to consider:
- the nicest user experience would be to direct the user to the support page in the same language (if possible)
- it is a content / link, that maybe should not be handed over to external translators
- maybe we could use 1 generic URL, and the website could be tweaked (if not already) to pick the best possible locale for the user automatically (based on browser settings etc). Alternatively we could pass the device locale as a query param to a generic URL, that could map to the best language server side, which might be more maintainable in the long run
- we could try to map the client (mobile / macOS) device selected language to the closest URL, and fall back to English, but that might become a maintenance overhead over time
- we could default the url to: `https://kiwix.org/get-involved/`, and not worry too much about it, and take it out from translations

@kelson42 What do you think ?

Reply:
Hi, I understand the considerations regarding the URL placement. I propose maybe utilizing the Localizable.strings file for each language.

For instance, I can add the link for each language directly into its respective Localizable.strings file, such as:

The `Support/es.lproj/Localizable.strings` file for the Spanish version would have the following content: `'Support_URL' = 'https://kiwix.org/es/get-involved/';`

This approach ensures that users are directed to the support page in their preferred language. If there's a language without corresponding support on the website, I'll include the English version URL in that file.

Would this approach even work?

Reply:
@BPerlakiH just use a static URL and let the website handle the rest.

Reply:
https://kiwix.org/get-involved/ to be clear

Reply:
This is clearly the job of the Website, one statis URL our side.